contracts/src/token/erc721/mod.rs

```diff
@@ -2498,11 +2498,11 @@ mod tests {
     #[motsu::test]
     fn interface_id() {
         let actual = <Erc721 as IErc721>::INTERFACE_ID;
-        let expected = 0x80ac58cd;
+        let expected = 0x80ac_58cd;
```

Comment:
This clippy recommendation I don't think worth it. May be we should better disable it cratewise

Reply:
Fine by me, @bidzyyys what do you say?

Reply:
Yep, let's revert and disable the rule.

Reply:
Reverted, but it seems there's no need to disable the rule as clippy does not catch this now :eyes: 
Maybe it's due to new a nightly version?